ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js

```diff
@@ -223,3 +228,7 @@ function templateCache($templateCache) {
     $templateCache.put(TEMPLATE_URL, template);
     $templateCache.put(TEMPLATE_MODAL_URL, modalTemplate);
 }
+
+var bundlize = (input) => input && input.split(/[^a-zA-Z0-9]+/).filter(x => x).join('-').toLowerCase();
```

Comment:
Why create this `var` (which should be `let`) ? The function should live within `bundlizeProvider` which you can then call directly within this file with `bundlizeProvider()(input);`. Actually, as better way would be to rely on the `$filter` service like so: `$filter('bundlize`)(input);`

Reply:
i didn't like `bundlizeProvider()(textToBundlize)`, but `$filter('bundlize')(text)` is not bad, changed